charts/devtron/README.md

```diff
@@ -24,32 +24,23 @@ It packages third party components like
 
 ### Install with Helm
 
-This chart is currently not available on the official helm repository therefore you need to download it to install it.
+Run the following command to install the latest version of Devtron along with the CI/CD module:
 
 ```bash
-$ git clone [https://github.com/devtron-labs/devtron-installation-script.git](https://github.com/devtron-labs/devtron-installation-script.git)
-$ cd devtron-installation-script/charts
-$ #modify values in values.yaml
-$ helm install devtron . -f values.yaml
+helm repo add devtron https://helm.devtron.ai 
+
+helm install devtron devtron/devtron-operator \
+--create-namespace --namespace devtroncd \
+--set installer.modules={cicd}
 ```
-For more details about configuration see the [helm chart configuration](#configuration)
 
-### Install with kubectl
+[Link to](https://docs.devtron.ai/install) more installation options for Devtron.
 
-If you don't want to install helm on your cluster and just want to use `kubectl` to install `devtron platform`, then please follow following steps:
+For more details about configuration see the [Helm chart configuration](#configuration)
```

Comment:
For more details on configuration, see the [Helm chart configuration](#configuration).

Reply:
done